tests/acceptance/features/coreApiVersions/fileVersions.feature

```diff
@@ -20,18 +20,12 @@ Feature: dav-versions
       | spaces           |
 
 
-  Scenario Outline: upload file and no version is available using various chunking methods (except new chunking)
-    Given using <dav-path-version> DAV path
-    When user "Alice" uploads file "filesForUpload/davtest.txt" to filenames based on "/davtest.txt" with all mechanisms except new chunking using the WebDAV API
+  Scenario: upload file and no version is available using various chunking methods
+    When user "Alice" uploads file "filesForUpload/davtest.txt" to filenames based on "/davtest.txt" with all mechanisms using the WebDAV API
     Then the HTTP status code of all upload responses should be "201"
-    And the version folder of file "/davtest.txt-olddav-regular" for user "Alice" should contain "0" elements
-    And the version folder of file "/davtest.txt-newdav-regular" for user "Alice" should contain "0" elements
-    And the version folder of file "/davtest.txt-olddav-oldchunking" for user "Alice" should contain "0" elements
-    Examples:
-      | dav-path-version |
-      | old              |
-      | new              |
-      | spaces           |
+    And the version folder of file "/davtest.txt-1dav-regular" for user "Alice" should contain "0" elements
+    And the version folder of file "/davtest.txt-2dav-regular" for user "Alice" should contain "0" elements
+    And the version folder of file "/davtest.txt-3dav-3chunking" for user "Alice" should contain "0" elements
 
```

Comment:
I am having second thoughts on this approach :thinking:

Reply:
Please let me know your concerns, and let's tackle them together.

Reply:
Yes please share.

Reply:
see https://github.com/owncloud/ocis/pull/10624/files#r1861492244